build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/DockerBase.java

```diff
@@ -18,7 +18,13 @@
     UBI("docker.elastic.co/ubi8/ubi-minimal:latest", "-ubi8"),
 
     // The Iron Bank base image is UBI (albeit hardened), but we are required to parameterize the Docker build
-    IRON_BANK("${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}", "-ironbank");
+    IRON_BANK("${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}", "-ironbank"),
+
+    // Base image with extras for Cloud
+    CLOUD("centos:8", "-cloud"),
+
+    // Based on CLOUD above, with more extras
+    CLOUD_ESS("centos:8", "-cloud-ess");
```

Comment:
Do we actually use this base image bit anywhere. I feel like we end up overriding this in `docker/build.gradle` anyhow.

Reply:
We reference `base.image` in a number of places in the docker `build.gradle`.

Reply:
Right, but we end up overwriting it [here](https://github.com/elastic/elasticsearch/pull/74980/files#diff-39ead019dc289b9b901bfa7df8efc01b1f040df71c88919b5fc7a6286b0d6597R382). Anyway to avoid that?

Reply:
Oh sorry, you mean in this specific case. Yes, we might as well not set it and comment why.